src/JungleSelect.js

```diff
@@ -285,10 +285,19 @@ class JungleSelect extends Component {
               if (item.get('filterable') === false) { return true }
               let matches = item
               let matching = searchableAttributes.map(k => {
-                let match = this.filterMethod(item.get(k))
-                if (match) {
-                  matches = matches.set(k, this.highlightFilterMatches(item.get(k)))
-                  this.highlights = this.highlights.set(index, matches)
+                let match = false
```

Comment:
Peut être utiliser que le `getIn/setIn` en forçant le searchable attribute en array.